### DISCUSSION TITLE
Where is No Results block?

### DISCUSSION THREAD

**Discussion by ri-co** [WordPress/gutenberg#42372] [Q&A] [answered]
Hey there, I'm currently working on a WP site with version 6.0 and I can't find No Results Block in Full Site Editor, in order to insert it in query loop editor. any help?

**Comment by carolinan** [comment, marked as answer]
Hi
The no results block can only be used inside the query loop. So you first need to make sure that you have selected the query loop, and that you are not inside for example the post template by mistake.

Some of the query loop variations also insert the block by default.
